api/drizzle/0002_lethal_dakota_north.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE "people" ALTER COLUMN "shop_id" SET NOT NULL;--> statement-breakpoint
+DO $$ BEGIN
+ ALTER TABLE "people" ADD CONSTRAINT "people_shop_id_shops_id_fk" FOREIGN KEY ("shop_id") REFERENCES "shops"("id") ON DELETE no action ON UPDATE no action;
+EXCEPTION
+ WHEN duplicate_object THEN null;
+END $$;
```

Comment:
Because we don't run migrations on Vercel automatically, we are ok with this change.